### DISCUSSION TITLE
Add a text to image

### DISCUSSION THREAD

**Discussion by Keither** [dlemstra/Magick.NET#1253] [Help] [answered]
Hi, i want to add a text to image like this

![aa](https://user-images.githubusercontent.com/5369977/189699222-dccf4eea-a7e4-4a2e-a39f-2b374fd3d7d7.jpg)
![bb](https://user-images.githubusercontent.com/5369977/189699237-1ed723b7-587a-4d8a-88b5-b16ed6908961.jpg)

anyone can help me, thanks!
sorry my English not good

**Comment by dlemstra** [comment, marked as answer]
You should probably first ask that question here: https://github.com/ImageMagick/ImageMagick/discussions. And then I can help you translate the command line example to C# if you need help to do that.

**Reply by Keither** [reply]
thanks!
